public/main.js

```diff
@@ -52,9 +52,30 @@ function reloadTodoList() {
         todos.forEach(function(todo) {
             var listItem = document.createElement("li");
             listItem.textContent = todo.title;
+            var deleteButton = document.createElement("button");
```

Comment:
I realise the rest of the code does this, but it might be worth adding a line above this one and after the last deleteButton.addEventListener - makes is easier to read :smile_cat: